### PR TITLE
Add card component

### LINKED ISSUE #87: Add Cards from Figma DL to Vocabulary
Blocked by: https://github.com/creativecommons/vocabulary/issues/86

**Description**
After the Cards are moved from Figma LT to Figma DL (documented in ticket above), we want to add each of the Cards to the Patterns section of the Vocabulary storybook. Note the names of the cards in Figma, and ensure those match as "Styles" in Vocabulary.

### PR DESCRIPTION
**Pull request by hugosolar**
Fixes #87

**Description**
This adds the cards component to storybook and also adds the `!important` property to background color to override the white background from Bulma's cards

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,281 @@
+$card-shadow : none;
+$card-content-padding: $space-small;
+$card-header-shadow: none;
+
+@import "../../node_modules/bulma/sass/components/card.sass";
```
Comment (brenoferreira) [inline src/styles/card.scss:5]: Did you try importing the sass file at the end and override here the Bulma variables https://bulma.io/documentation/components/card/#variables?

File: src/stories/cards.stories.js [hunk 2]
```diff
@@ -0,0 +1,300 @@
+import { withKnobs, select } from '@storybook/addon-knobs'
+export default { title: 'Cards', decorators: [withKnobs] }
+
+export const PostVertical = () => {
+  return `
+    <style>
+      .card {
```
Comment (brenoferreira) [inline src/stories/cards.stories.js]: Shouldn't each card type define it's own default `width`?  Users could override it if they want to

Comment (hugosolar) [reply src/stories/cards.stories.js]: The idea is the card should adapt to the container, working this way is easier to handle responsive views

File: src/styles/card.scss (hunk 1 above)
Comment (hugosolar) [reply src/styles/card.scss:5]: Ok, this was my mistake... there's no need to use `!important`

Comment (hugosolar) [reply src/styles/card.scss:5]: It's fixed in the last commit

File: src/stories/cards.stories.js (hunk 2 above)
Comment (brenoferreira) [reply src/stories/cards.stories.js]: OK. So, given that the code here is part of the documentation, do you think it's better to change it to something like:

```
<div style="width: 400px">
  <article class="card ....
    .....
</div>
```

I think that makes it more clear to users of the library that the `.card` parent container should have a width set, not the `.card` element itself, from what I understand

Comment (hugosolar) [reply src/stories/cards.stories.js]: Documentation will be written using docs plugin and the source of the story shouldn't be used as documentation. is it really necessary?

Comment (hugosolar) [reply src/stories/cards.stories.js]: Ok, now it's updated with your requested changes... I'll leave the MDX documentation for later

**Review by brenoferreira — APPROVED** [review]

**Pull request merged**